### ISSUE TITLE
ALEInterface::isSupportedROM issue? Rom falsely not supported

### ISSUE THREAD

**Issue by BenTNorman** [mgbellemare/Arcade-Learning-Environment#433]
Hi, I have been trying to get OpenAi gym to work with pong, and apparently this requires ale-py=0.7.0, rather than the current ale-py 0.71, which is the version installed using pip gym[atari].

Once changing version, I ​sourced the relevant pong.bin file, and it has the correct check sum, as listed in the md5.txt file (60E0EA3CBE0913D39803477945E9E5EC)

It can be loaded fine, with no warnings using ale.loadROM("roms/pong.bin")
However ALEInterface.isSupportedROM("roms/pong.bin") returns None, and ale-import-roms does not register it as supported.

**Comment by BenTNorman** [comment]
Windows 10
Fresh anaconda install.

Fyi this is what is displayed on ale.loadROM:
Game console created:
  ROM file:  roms/pong.bin
  Cart Name: Video Olympics (1978) (Atari)
  Cart MD5:  60e0ea3cbe0913d39803477945e9e5ec
  Display Format:  AUTO-DETECT ==> NTSC
  ROM Size:        2048
  Bankswitch Type: AUTO-DETECT ==> 2K

Running ROM file...
Random seed is 1633550324

**Comment by JesseFarebro** [comment]
And you're manually testing `ALEInterface.isSupportedROM`? Can you tell me what output you get when running the following:

```py
from pathlib import Path
from ale_py import ALEInterface

rom = "roms/pong.bin"

print(ALEInterface.isSupportedROM(rom))
print(ALEInterface.isSupportedROM(Path(rom))
```

**Comment by BenTNorman** [comment]
None
None

**Comment by JesseFarebro** [comment]
This issue is now fixed as of `v0.7.2` which is now published on PyPI.
